tests/inference_service_utils/http_inference_service_setup.py

```diff
@@ -50,7 +50,7 @@ async def inference(inp: Request):
       service_name = inp.url.path.split('/')[-1]
       inp = await inp.json()
       df = name_to_df[service_name]
-
+      df = df.fillna('Missing')
```

Comment:
Some values are missing, causing the entire row to be dropped. As a result, the AIDB results differ from the ground truth.

Reply:
This doesn't seem like the correct solution

Reply:
What if the value is "missing"?